bugbug/models/accessibility.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import logging
+
+import xgboost
+from imblearn.over_sampling import BorderlineSMOTE
+from imblearn.pipeline import Pipeline as ImblearnPipeline
+from sklearn.compose import ColumnTransformer
+from sklearn.feature_extraction import DictVectorizer
+from sklearn.pipeline import Pipeline
+
+from bugbug import bug_features, bugzilla, feature_cleanup, utils
+from bugbug.model import BugModel
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+class AccessibilityModel(BugModel):
+    def __init__(self, lemmatization=False):
+        BugModel.__init__(self, lemmatization)
+
+        self.calculate_importance = False
+
+        feature_extractors = [  # TODO: Check effect of adding/removing some features
+            bug_features.HasSTR(),
+            bug_features.Severity(),
+            bug_features.Keywords(),
+            bug_features.Whiteboard(),
+            bug_features.EverAffected(),
+            bug_features.AffectedThenUnaffected(),
+            bug_features.HasImageAttachment(),
+            bug_features.HasImageAttachmentAtBugCreation(),
```

Comment:
There is no need for both features in this context. They are the same; the only difference is that the second filters out attachments that were uploaded after creating the bug. Since we will be rolling back the bugs to the creation time, both should always have the same exact value.